src/Apps/CollectorProfile/Routes/Saves/Components/OfferSettingsModal/__tests__/OfferSettingsModal.jest.tsx

```diff
@@ -1,13 +1,20 @@
-import { screen, fireEvent, waitFor } from "@testing-library/react"
+import { screen, fireEvent } from "@testing-library/react"
 import { useMutation } from "Utils/Hooks/useMutation"
 import { useSystemContext } from "System/SystemContext"
 import { render } from "DevTools/renderWithMockBoot"
 import { OfferSettingsModal } from "Apps/CollectorProfile/Routes/Saves/Components/OfferSettingsModal/OfferSettingsModal"
 import { CollectorProfileSavesRoute_me$data } from "__generated__/CollectorProfileSavesRoute_me.graphql"
 import { createMockEnvironment } from "relay-test-utils"
+import { flushPromiseQueue } from "DevTools/flushPromiseQueue"
 
 jest.mock("Utils/Hooks/useMutation")
 jest.mock("System/useSystemContext")
+jest.mock(
+  "Apps/CollectorProfile/Routes/Saves/Components/OfferSettingsModal/OfferSettingsListItem",
+  () => {
+    return { OfferSettingsListItemFragmentContainer: () => <div /> }
```

Comment:
The modal renders a fragment container, and we would need a query renderer to make it work. Otherwise, I got an error:

`Relay(OfferSettingsListItem)` tried to render a context that was not valid this means that `Relay(OfferSettingsListItem)` was rendered outside of a query renderer.

Since the tests don't care about rendering the items, I just mock it. However, if we want to test toggling privacy for lists, we'll probably need to use a query render to make it work.